cms/djangoapps/contentstore/tests/test_course_settings.py

```diff
@@ -905,48 +905,21 @@ def test_advanced_components_munge_tabs(self):
         """
         Test that adding and removing specific advanced components adds and removes tabs.
         """
-        open_ended_tab = {"type": "open_ended", "name": "Open Ended Panel"}
```

Comment:
So as someone who worked on tabs semi-recently, I believe that with the removal of ORA1, notes is the only tab that will be added this way, and it's not really a recommended pattern, so I think it's okay that we're only testing the presence of one tab this way.